### ISSUE TITLE
[NEW] documentation cmds not part of redis

### ISSUE THREAD

**Issue by raphaelauv** [Snapchat/KeyDB#424]
**The problem/use-case that the feature addresses**

the documentation do not explictit the cmds not part of redis

like `hrename`

**Description of the feature**

A documentation page of feature of keyDB not part of redis

**Comment by benschermel** [comment]
I agree, this is a good idea to consolidate list in one place. We will work on this. Transferring the issue to our documentation repo
